packages/wrangler/src/cloudchamber/apply.ts

```diff
@@ -108,12 +109,46 @@ function applicationToCreateApplication(
 	return app;
 }
 
+function observabilityToConfiguration(
+	observability: Observability | undefined,
+	existingObservabilityConfig: ObservabilityConfiguration | undefined
+): ObservabilityConfiguration | undefined {
+	const observabilityLogsEnabled =
+		observability?.logs?.enabled === true ||
+		(observability?.enabled === true && observability?.logs?.enabled !== false);
+	if (observabilityLogsEnabled) {
+		return { logs: { enabled: true } };
+	}
+
+	const observabilityLogsDisabled =
+		observability?.logs?.enabled === false ||
+		(observability?.enabled === false && observability?.logs?.enabled !== true);
+	if (observabilityLogsDisabled) {
+		return { logs: { enabled: false } };
+	}
+
+	if (existingObservabilityConfig !== undefined) {
+		return { logs: { enabled: false } };
+	}
+
+	return undefined;
```

Comment:
```suggestion
    const observabilityEnabled = (observability?.enabled || (observability?.logs?.enabled))
    if (observabilityEnabled !== undefined){
       return { logs: { enabled: observabilityEnabled } };
    }
	if (existingObservabilityConfig !== undefined) {
		return { logs: { enabled: false } };
	}
	return undefined;
```

but also I think I'm missing a bit of context on `existingObservabilityConfig` - could existingObservabilityConfig be true? why are we forcing it to false? 
is the idea if observability was set previously, but its not set in the current deployment, you want to default to false?  

if i've understood that correctly, +1 to Cody's suggestion to default to false if undefined, and moving this config validation :)